src/m365/spo/commands/list/list-get.ts

```diff
@@ -101,17 +100,16 @@ class SpoListGetCommand extends SpoCommand {
     );
   }
 
-  #initOptionSets(): void {
-    this.optionSets.push({ options: ['id', 'title', 'url'] });
-  }
-
   public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
     if (this.verbose) {
       await logger.logToStderr(`Retrieving information for list in site at ${args.options.webUrl}...`);
     }
 
     let requestUrl: string = `${args.options.webUrl}/_api/web/`;
 
+    if (!args.options.id && !args.options.title && !args.options.url) {
```

Comment:
Instead of the long condition here, we could just convert this to an `else` condition in the condition chain below.

Reply:
Done.